waiter/config-full.edn

```diff
@@ -484,6 +484,14 @@
                            ;; The number of hosts on which health checks must fail in order to declare a deployment error:
                            :min-hosts 2}
 
+ :health-check-config {
+                       ;; The HTTP connect timeout and idle timeout (milliseconds) for instance health checks:
+                       :health-check-timeout-ms 200
+
+                       ;; Number of allowed health check failures before unhealthy instances are
+                       ;; flagged as never-passed-health-checks upon failure
+                       :max-failed-health-checks 5}
```

Comment:
The name seems off to me.  `min-failed-health-checks` seems equally appropriate, for example.